cds_dojson/marc21/fields/videos/video.py

```diff
@@ -140,9 +141,21 @@ def location(self, key, value):
 
 
 @model.over('internal_note', '^595__')
+@ignore_value
 def internal_note(self, key, value):
     """Internal note."""
-    return ", ".join(filter(None, [value.get('a'), value.get('s')]))
+    _internal_categories = defaultdict(list)
+    _internal_categories.update(self.get('internal_categories', {}))
+    _internal_notes = self.get('internal_note', '').splitlines()
+    for v in force_list(value):
```

Comment:
silly question, why the `force_list`?

Reply:
Depending on how the MARC21 is defined value could be a single element or a list, like that we are sure is always a list

This is one element
```
<datafield tag="595" ind1=" " ind2=" ">
  <subfield code="a">test1</subfield>
</datafield>
```
This is a list
```
<datafield tag="595" ind1=" " ind2=" ">
  <subfield code="a">test1</subfield>
  <subfield code="a">test2</subfield>
</datafield>
```